src/Form/Layout.php

```diff
@@ -209,8 +210,10 @@ public function recursiveRender()
             $template->trySet('label_for', $el->id . '_input');
             $template->set('control_class', $el->getControlClass());
 
-            // backward compatibility - will be removed in dec-2020
-            $template->trySet('field_class', $el->getControlClass());
+            // BC-break exception
```

Comment:
missing clear date when it should be remove

Reply:
I just added it but problem is later if we remove and the dev template contains region `field_class` it will just not work without warning